DSharpPlus/Entities/Interaction/Application/DiscordApplicationCommand.cs

```diff
@@ -140,6 +148,14 @@ public DiscordApplicationCommand(string name, string description, IEnumerable<Di
             this.DefaultMemberPermissions = defaultMemberPermissions;
         }
 
+        /// <summary>
+        /// Gets the mention string for a subcommand.
+        /// </summary>
+        /// <param name="name">The name of the subcommand.</param>
+        /// <returns>The mention string for the subcommand.</returns>
+        public string GetSubcommandMention(string name)
+            => $"{this.Name} {name}:{this.Id}";
```

Comment:
One big thing you're missing is the fact that subcommands can be multiple levels (I believe 2 levels) deep. This may work, but I think it's a better idea to at least add a check whether the requested subcommand is valid.

Reply:
idea; `params string[]`

Reply:
https://discord.com/developers/docs/interactions/application-commands#subcommands-and-subcommand-groups

Reply:
> idea; `params string[]`

No pls

Reply:
What should be done if the requested subcommand isn't valid?
Also, I noticed I was missing the <>s, so I'll fix that too

Reply:
> > idea; `params string[]`
> 
> No pls

instead maybe next time propose a proper alternative instead of just "no"

Reply:
Also, subcommands cannot be nested.

Reply:
Well, technically speaking they can by 1 level, but even then Discord's markdown is strange.